Android/build.gradle

```diff
@@ -1,6 +1,6 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id 'com.android.application' version '8.0.2' apply false
-    id 'com.android.library' version '8.0.2' apply false
+    id 'com.android.application' version '7.4.2' apply false
```

Comment:
Why downgrade this?

Reply:
This one is compatible with my android studio

Reply:
Revert this for now and keep these changes on your local. Send the error you're facing and I'll try to fix it.

Reply:
What do you mean by revert?

Reply:
Nvm leave it just complete rest of the changes

Reply:
@07jasjeet I made all the other changes

Reply:
@essy16 please recheck there are still comments to be resolved above.

Reply:
@07jasjeet i made the recent commit and I did all those ...please check

Reply:
@07jasjeet i made the recent commit and I did all those ...please check